app/AppInit.jsx

```diff
@@ -50,11 +50,30 @@ class AppInit extends React.Component {
             apiConnected: false,
             apiError: false,
             syncError: null,
-            status: ""
+            status: "",
+            extendeLogText: []
         };
+        this.saveExtendedLog = this.saveExtendedLog.bind(this);
+    }
+
+    componentDidCatch(error) {
+        this.saveExtendedLog(error.stack);
     }
 
     componentWillMount() {
+        var thiz = this;
+        console.log = function() {
+            thiz.saveExtendedLog(arguments[0]);
```

Comment:
Why do we save only first argument? Would this work with something like that:
`console.log("Some info: ", someInfo)`